### ISSUE TITLE
Update Packagist?

### ISSUE THREAD

**Issue by ggedde** [fennb/phirehose#117]
Could you update Packagist with the latest version?
Thanks

**Comment by fennb** [comment]
If someone would like to submit a pull request doing the right thing I'm happy to review! (sorry, don't know Packagist myself and what the ins/outs are of changing versions).

**Comment by ggedde** [comment]
I was just referring to updating Composer (Packagist)
https://packagist.org/packages/fennb/phirehose

Not a big deal as I just used `composer require fennb/phirehose:dev-master` to get the latest.

I figured you created the package there?
If you have Packagist setup to auto pull from GitHub then you just need to create a new Tag on Github.

Packagist was showing version 1.0.1 as the latest which was last updated in 2014
